### ISSUE TITLE
iOS: Modal is not automatically focused, and you can focus things outside the modal

### ISSUE THREAD

**Issue by simonflk** [GeekyAnts/NativeBase#4986]
### Description

Using Voiceover on iOS. When I open a modal I expect the focus to shift to the modal so that I know something happened. I should be able to navigate the contents of the modal clearly and not be able to access content from outside the modal while the modal is open

### CodeSandbox/Snack link

https://snack.expo.dev/@simonflk/nativebase-voiceover-modal-issue

### Steps to reproduce

1. Open the snack link
2. Enable Voiceover on iOS
3. focus the button
4. Double-tap to press the button
5. The modal is open, but the trigger button is still focused
6. Swipe left or right to access the contents of the modal

### NativeBase Version

3.4.3

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

### Additional Information

iOS 15.4

https://user-images.githubusercontent.com/4867329/166302548-1a9d075f-9b56-4d66-a398-8674031eee66.mov

**Comment by Viraj-10** [comment]
Hi @simonflk, you can try [this](https://snack.expo.dev/@viraj_10/modal-accessibility-) workaround to shift the `focus` to modal. We will fix this in the next sprint.

**Comment by bahri-dev** [comment]
The same issue is reproduceable in UWP. Outside the modal should be impossible to focus.
Is it fixed now?
